readme.md

```diff
@@ -0,0 +1,22 @@
+#Tunnetler#
+
+Is a revival of old-school computer game, called Tunneler (You can play this at: https://classicreload.com/tunneler.html). Major difference of Tunnetler in comparison to it's precceder will be network multiplayer. We dont want to bring Tunneler from the dead, we want to enhance it. So there are possible any reinforcements, which will make Tunnetler even more fun!
+
+##Technical info##
+
+Tunnetler is a desktop game created on Rust meetups in Bratislava's Progressbar. It is programmed exclusively in Rust language. We using Rust Piston game engine and various other Rust and Piston related crates.
+
+##Contribution##
+
+If you want to contribute, keep in mind following:
+
+- We are contributing using Pull Requests, which will be reviewed by Kixunil, and then approved or commented.
+- Any PR must be based upon existing Issue.
+- If you have any ideas or suggestions for further development, create an Issue with propper description. It will be discussed.
+- Ideally, your commit messages should be like "subject description: specific change description". So we have an idea what your commit contains.
+- This project is OpenSource, which means, your code must be commented, and be easy to read. 
+- We are working on this project because we want to learn Rust. If you have any rust-related questions, feel free to ask. We will perhaps create RustWiki later.
+
+For any further questions, please contact us on:
+
+matej.srubar@exile.sk
```

Comment:
Perhaps it is. What do you suggests? I dont quite know how to leave github contact or something, which is assesible also with users, whose does not have an GitHub account.